src/System.Numerics.Vectors/ref/System.Numerics.Vectors.cs

```diff
@@ -305,6 +305,9 @@ public partial struct Vector<T> : System.IEquatable<System.Numerics.Vector<T>>,
         public Vector(T value) { throw null; }
         public Vector(T[] values) { throw null; }
         public Vector(T[] values, int index) { throw null; }
+#if netcoreapp
```

Comment:
Typically we put these into separate `.netcoreapp.cs` files instead of putting `#if` in the code.  See https://github.com/dotnet/corefx/blob/master/src/System.Net.Sockets/ref/System.Net.Sockets.netcoreapp.cs for an example.